src/modules/auth/credentials/CredentialsDetailContainer.js

```diff
@@ -487,8 +487,8 @@ function CredentialsInfoCard(props) {
 				{editMode ?
 					<React.Fragment>
 						<ButtonGroup>
-							<Button color="primary" type="submit">{t("Save")}</Button>
-							<Button color="outline-primary" type="button" onClick={(e) => (setEditMode(false), setOnUpdate(false))}>{t("Cancel")}</Button>
+							<Button color="primary" disabled={errors?.phone?.message} type="submit">{t("Save")}</Button>
```

Comment:
@Samsunggora I understand that if there will be error on phone, the button will be disabled, but what if there will be an error on both or just on email? You need to cover those cases as well